src/descriptor/mod.rs

```diff
@@ -607,39 +479,39 @@ impl<Pk: MiniscriptKey> DescriptorTrait<Pk> for Descriptor<Pk> {
         }
     }
 
+    /// Attempts to produce a non-malleable satisfying witness and scriptSig to spend an
+    /// output controlled by the given descriptor; add the data to a given
+    /// `TxIn` output.
+    pub fn satisfy<S>(&self, txin: &mut TxIn, satisfier: S) -> Result<(), Error>
+    where
+        S: Satisfier<Pk>,
+    {
+        // easy default implementation
```

Comment:
I think this comment may have been here because this used to be in a trait and could have been overridden. That is no longer possible now so perhaps remove this?

Reply:
woops yep, thanks